kernel/dreadnought/src/task.rs

```diff
@@ -91,6 +91,7 @@ pub type Result<T> = core::result::Result<T, Error>;
 pub enum Error {
     Cancelled,
     Panic(PanicInfoOwned),
-    Reaped,
+    /// A `JoinError` should not occur; this indicates a BUG in Theseus's task mgmt.
+    JoinError(&'static str),
```

Comment:
Should probably be `Join`, not `JoinError`.

<https://rust-lang.github.io/rust-clippy/master/#enum_variant_names>

Reply:
good point.